katalog/fluentd/fluent.conf

```diff
@@ -71,11 +70,10 @@
      queue_limit_length "#{ENV['FLUENT_ELASTICSEARCH_BUFFER_QUEUE_LIMIT_LENGTH'] || '32'}"
      retry_max_interval "#{ENV['FLUENT_ELASTICSEARCH_BUFFER_RETRY_MAX_INTERVAL'] || '30'}"
      retry_forever true
+     overflow_action drop_oldest_chunk
```

Comment:
Any reason for not setting block? (doc [here](https://docs.fluentd.org/configuration/buffer-section), example  [here](https://github.com/helm/charts/blob/master/stable/fluentd/values.yaml#L149)

Reply:
I thought "drop_oldest_chunk" could help when system is overflowed with logs, but if "block" helps in the same way i can change to it.

I think fluentbit will complain about being unable to send logs to fluentd right?

Reply:
block should backpressure to the input plugin instead of dropping logs, not sure how it is propagated to fluentbit

Reply:
https://fluentbit.io/documentation/0.13/configuration/backpressure.html here the doc

Reply:
Changed to block, and in fluent-bit there is infinite retry, so i think it's ok!